decidim-generators/Gemfile

```diff
@@ -29,4 +29,11 @@ group :development do
   gem "spring", "~> 2.0"
   gem "spring-watcher-listen", "~> 2.0"
   gem "web-console", "~> 3.5"
+
+  # Profiling gems
```

Comment:
Is it really necessary to have it on the decidim-generators? For me there are two kind of developers of Decidim: 
* The ones that only want to use the generated app (aka "decidim implementers") 
* The ones that want to fix bugs / make new features for the decidim gem (aka  "decidim developers") 

I see the value on having this for developers, but probably for implementers is a little too much information, specially seeing that lots of implementers is their first time working with a Ruby on Rails project and the bullet bullet/profiling tools could be scary if what you're trying to see is "only" a fresh Decidim installation